File: frontend/src/features/actividades/recibir_instruccion_pago/models/recibir_instruccion_pago.ts
export interface RecibirInstruccionPago {
  id_recibir_instruccion_pago: number;
  id_expediente: number;
  enviar_a_reparo: boolean;
  condicion_especial_desembolso?: string | null;
  observaciones?: string | null;

  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by?: number | null;
  modified_date?: string | null;
}

File: backend/Multibanca.Domain.Models/Multibanca/recibir_instruccion_pago.cs
using Common.Domain.Models;
using System;

namespace Multibanca.Domain.Models.Multibanca
{
    public class recibir_instruccion_pago : base_auditoria
    {
        public int id_recibir_instruccion_pago { get; set; }
        public long id_expediente { get; set; }
        public bool? enviar_a_reparo { get; set; }
        public string? condicion_especial_desembolso { get; set; }
        public string? observaciones { get; set; }
    }
}
